src/features/BroadcastTransaction/components/BroadcastTx.tsx

```diff
@@ -98,20 +94,25 @@ interface Props {
   handleNetworkChanged(network: NetworkId): void;
 }
 
-const BroadcastTx = ({ signedTx, network, onComplete, handleNetworkChanged }: Props) => {
+const BroadcastTx = ({
+  signedTx,
+  network: networkId,
```

Comment:
Can you change this elsewhere to just be `networkId` so we don't confuse ourselves in the future?